src/CommunityPatch/Patches/EarlyStoryVisibleTimeoutPatch.cs

```diff
@@ -82,14 +87,14 @@ private static bool IsFirstStoryPhase()
         && SecondPhaseInstance == null;
 
     private static void SetStoryVisibleTimeoutIfNeeded(QuestBase quest) {
-      if (!IsFirstStoryPhase() || !quest.IsSpecialQuest || !quest.IsOngoing)
+      if (FirstPhaseTimeLimitInYears == null || !IsFirstStoryPhase() || !quest.IsSpecialQuest || !quest.IsOngoing)
```

Comment:
Avoid changing the quests at all if we couldn't get the time limit value. Otherwise it would have been 0 and we would have insta-cancelled all the quests.